src/cgame/cg_draw.cpp

```diff
@@ -2194,30 +2197,66 @@ static void CG_DrawVote(void) {
       }
     }
 
-    if (!(cg.snap->ps.eFlags & EF_VOTED)) {
-      line_a = ETJump::stringFormat("VOTE(%i): %s", sec, cgs.voteString);
+    const bool canVote = cgs.clientinfo[cg.clientNum].team != TEAM_SPECTATOR ||
+                         etj_spectatorVote.integer;
+    const bool isRtvVote = rtvHandler->rtvVoteActive();
+    const int rtvYesVotes = rtvHandler->getRtvYesVotes();
 
-      if (cgs.clientinfo[cg.clientNum].team == TEAM_SPECTATOR &&
-          etj_spectatorVote.integer <= 0) {
-        line_b = ETJump::stringFormat("YES:%i, NO:%i (%s)", cgs.voteYes,
-                                      cgs.voteNo, "Spectators can't vote");
+    if (isRtvVote) {
+      if (!(cg.snap->ps.eFlags & EF_VOTED)) {
+        line_a = ETJump::stringFormat("VOTE(%i): %s", sec, cgs.voteString);
+        line_b = ETJump::stringFormat(
+            "Change map(%s):%i, Keep current map(%s):%i%s", str1, rtvYesVotes,
+            str2, cgs.voteNo, canVote ? "" : " (Spectators can't vote)");
       } else {
-        line_b = ETJump::stringFormat("YES(%s):%i, NO(%s):%i", str1,
-                                      cgs.voteYes, str2, cgs.voteNo);
+        line_a =
+            ETJump::stringFormat("(%i) YOU VOTED ON: %s", sec, cgs.voteString);
+        line_b = ETJump::stringFormat("Change map:%i, Keep current map:%i",
+                                      rtvYesVotes, cgs.voteNo);
+
+        x_b = 13;
+
+        if (cgs.votedYes) {
+          const char *yesVotes = "Change map";
+          auto strWidth =
+              static_cast<float>(ETJump::DrawStringWidth(yesVotes, 0.23f));
+          CG_DrawRect_FixedBorder(x_b - 2, 214 - 10 + 12, strWidth + 4, 12, 1,
+                                  color);
+        } else if (cgs.votedNo) {
+          std::string yesVotes =
```

Comment:
I generally prefer using explicit type, the reason why `auto` is used for the next variable after this is because it's assigned via `static_cast`, in which case it makes sense imo to use `auto` instead of repeating the type twice.